tailwind.config.ts

```diff
@@ -89,9 +89,11 @@ const config: Config = {
         "dark-blue": "#09328b",
         "text-blue": "#224ba4",
         "green-10": "#E9FBFE",
+        "deep-blue": "#082B76",
```

Comment:
**suggestion (blocking):** Can this be removed?  That color is already part of the `navy` colorset in the Tailwind config.